python/lsst/ts/ATWhiteLightSource/chillerComponent.py

```diff
@@ -43,10 +45,12 @@ async def send_command(self, cmd):
         if self.connected:
             async with self.chiller_com_lock:
                 self.writer.write(cmd)
-                response = await asyncio.wait_for(self.reader.readuntil(separator=b'\r'), timeout=5)
+                response = await asyncio.wait_for(
+                    self.reader.readuntil(separator=b"\r"), timeout=5
+                )
```

Comment:
Also note that the doc strings for the ChillerPacketEncoder methods that create commands all say "Generates the ascii string", which is incorrect, because these methods return a `bytes` not a `str`.

To fix this I suggest you move the encoding from `_commandWrapper` to `send_command` (a simple change) so that the doc strings are correct. But fixing all the doc strings is certainly another option.